.github/workflows/cypress.yaml

```diff
@@ -22,20 +22,28 @@ jobs:
         run: |
           pip install poetry
           poetry install
-      - name: set opscenter config
+      - name: set config
         env:
           SNOWFLAKE_ACCOUNT: ${{ secrets.SNOWFLAKE_ACCOUNT }}
           SNOWFLAKE_USERNAME: ${{ secrets.SNOWFLAKE_USERNAME }}
           SNOWFLAKE_PASSWORD: ${{ secrets.SNOWFLAKE_PASSWORD }}
         run: |
           mkdir $HOME/.snowsql
           envsubst < deploy/opscenter.config > $HOME/.snowsql/config
-      - name: set opscenter env
+      - name: set env
         run: |
-          echo "OPSCENTER_PACKAGE=PACKAGE_DEV" >> "$GITHUB_ENV"
-          echo "OPSCENTER_DATABASE=\"DATABASE_OPSCENTER_DEV\"" >> "$GITHUB_ENV"
-          echo "OPSCENTER_APP=APP_OPSCENTER_DEV" >> "$GITHUB_ENV"
-      - name: run opscenter streamlit
+          suffix=$(echo ${GITHUB_HEAD_REF} | sed "s/\./_/g" | sed "s/-/_/g" | sed "s\/\_\g")_${GITHUB_RUN_ID}
+          echo "OPSCENTER_PACKAGE=PACKAGE_${suffix}" >> "$GITHUB_ENV"
+          echo "OPSCENTER_DATABASE=\"DATABASE_${suffix}\"" >> "$GITHUB_ENV"
+          echo "OPSCENTER_APP=APP_${suffix}" >> "$GITHUB_ENV"
+      - name: run deploy
+        env:
+          OPSCENTER_PACKAGE: ${{env.OPSCENTER_PACKAGE}}
```

Comment:
I don't think any of these env variables are used in devdeploy... @joshelser what do you think?

Reply:
Correct, they're not used in devdeploy which is what lead to some confusion.

This is the next level of cleanup i'd do between deploy.py and devdeploy.py (and intentionally avoided in case we needed to do any urgent 'hotfixes' of opscenter). We should be able to combine some of these functions now that we have these two entrypoints closer together.